app/src/main/java/com/yacgroup/yacguide/AboutActivity.kt

```diff
@@ -43,6 +43,11 @@ class AboutActivity : BaseNavigationActivity() {
         _createEntry(getString(R.string.privacy_policy), callback = { _showPrivacyPolicy() })
         _createEntry(getString(R.string.source_code_and_support), getString(R.string.github_url))
         _createEntry(getString(R.string.license), getString(R.string.license_url))
+        _createEntry(
```

Comment:
Might it make sense to migrate this view to RecyclerView using `BaseViewItemAdapter`, too? If yes, you may create a ticket for that.

Reply:
I will have a look. Thanks for the hint.